### ISSUE TITLE
change MCMC chain length and sampling

### ISSUE THREAD

**Issue by taylori** [cgrandin/hake-assessment#347]
**Used in 2017**

    ss3 -mcmc 12000000 -mcsave 10000 -mcseed 36519
    ss3 -mceval
The 12 million samples saving 1 in every 10 thousand results in 1200 samples saved.
These values are then been combined with starter.ss values of

    201     # MCMC burn-in
    1       # MCMC thinning interval
For 999 samples total.

**Proposed for 2018**
    ss3 -mcmc 24000000 -mcsave 10000 -mcseed 36519
    ss3 -mceval
The 24 million samples saving 1 in every 10 thousand results in 2400 samples saved.
These values are then been combined with starter.ss values of

    400     # MCMC burn-in
    1       # MCMC thinning interval

to get 2000 total samples

**Comment by taylori** [comment]
Updated README.md to reflect the new MCMC commands. 

I started separate chains on copies of model 2018.40 with both the old (12 million) and new (24 million) commands to get a longer chain while also having a shorter one to see preliminary results earlier. 

Closing this issue. If there's any problem with the MCMCs, or evidence of lack of convergence, we can re-open.

**Comment by taylori** [comment]
I know that I closed this issue already, but just an update that there are now mcmc samples for a chain of length 12 million (and the old thinning burn-in to get 999 samples) uploaded to Google drive in a folder called "mcmc_12mil" within model 2018.40_base_model. The 24-million chain should be done by tomorrow afternoon which I will upload to a new "mcmc" folder.

**Comment by cgrandin** [comment]
I ran a 24 million chain and uploaded it to the base model to compare if necessary. It's called `mcmc-24mil-grandin`

**Comment by taylori** [comment]
Thanks Chris. I did the math wrong on mine, and it finished at 6am today rather than taking until this afternoon. I just checked and the posteriors.sso files were identical between the two, so let's just use yours, renaming the directory (either on Google Drive or after downloading) to "mcmc".

For future reference, my total run time (as reported after -mcmc is complete, and shown in difference between date modified for ss.mcmc and ss.psv, except that Google Drive messes up the date modified).
